charts/victoria-logs-single/values.yaml

```diff
@@ -346,7 +346,7 @@ vector:
           headers:
             VL-Time-Field: timestamp
             VL-Stream-Fields: stream,kubernetes.pod_name,kubernetes.container_name,kubernetes.pod_namespace
-            VL-Msg-Field: message,msg,_msg,log.msg,log
+            VL-Msg-Field: message,msg,_msg,log.msg,log.message,log
```

Comment:
Could you add this change to a changelog as well?

Reply:
added